client/src/lib/api/viame.service.ts

```diff
@@ -18,6 +18,13 @@ interface Pipe {
   pipe: string;
   type: string;
 }
+
+export interface Types {
+  video: [string];
+  image: [string];
+  web: [string];
```

Comment:
`string[]` ?

I belive `[string]` indicates an array with 1 element of type string.